open_humans/templates/member/my-member-settings.html

```diff
@@ -116,7 +116,7 @@ <h5>Allow other members to contact you?</h5>
     <div class="col-md-6">
       <p>Use external accounts to login to Open Humans</p>
 
-      <a href="{% url 'socialaccount_connections' %}?next={% url 'my-member-settings' %}"
```

Comment:
I don't think this should be removed, it seems to be breaking this:
https://github.com/OpenHumans/open-humans/blob/master/open_humans/member_views.py#L190